internal/services/contracts_events_consumer.go

```diff
@@ -200,6 +200,17 @@ func (c *ContractsEventsConsumer) handleVehicleTransfer(e *ContractEventData) er
 
 	defer tx.Rollback() //nolint
 
+	rowsAff, err := models.NFTPrivileges(
+		models.NFTPrivilegeWhere.TokenID.EQ(types.Decimal(tkID)),
+		models.NFTPrivilegeWhere.UserAddress.EQ(args.To.Bytes()),
```

Comment:
I don't think we want any filter on `user_address`, all privileges must go. The "`to`" is the only one who will have privileges when this is over—of course, he has them implicitly.